test/e2e/util.go

```diff
@@ -80,6 +80,33 @@ func makeBackup(f *framework.Framework, clusterName string) error {
 	return cluster.RequestBackupNow(f.KubeClient.Client, addr)
 }
 
+func checkBackupDeleted(f *framework.Framework, clusterName string) error {
+	time.Sleep(5 * time.Second)
```

Comment:
can you add a simple for loop to check every 1 seconds for 60seconds. i kind of hate this random wait.